upt_macports/upt_macports.py

```diff
@@ -189,6 +201,15 @@ def _cpandir(self):
             self.logger.info('Using fallback location for dist file')
             return f' ../../authors/id/{fallback_dist}/'
 
+    @property
+    def homepage(self):
+        homepage = self.upt_pkg.homepage
+        portgroup_default = 'metacpan.org/pod'
+        if homepage.startswith('http') and portgroup_default not in homepage:
+            return homepage
+        else:
+            return None
```

Comment:
Why not use metacpan.org/release/{pkg} as the default value?

Reply:
well, that's not the value that is used as default in the Perl portgroup; so it if we decide this should be the default it will have to be changed there as well.